product.php

```diff
@@ -0,0 +1,58 @@
+<?php
+require_once 'src/Services/ProductsDisplayService.php';
+require_once 'src/Models/ProductModel.php';
+require_once 'src/Factory/FurnitureDatabaseConnector.php';
+require_once 'src/Entities/ProductEntity.php';
+require_once 'src/Entities/IndividualProductEntity.php';
+
+$db = furnitureDatabaseConnector::connect();
```

Comment:
is this going to work?
```suggestion
$db = FurnitureDatabaseConnector::connect();
```